1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 77 Line 20_
```markdown
Names are case sensitive, i.e. `case_sensitive`, `CASE_SENSITIVE`, and `Case_Sensitive` are all different.

#### Variable naming style
One letter characters such as `a`, `b`, and `c` are too short and not at all descriptive (in general) for meaningful variable names.

On the other hand, something like `number_of_particles_in_target_region` is too long;

A better balance might be `num_target_particles`. Perhaps instead of underscore, we prefer camel case i.e. `numTargetParticles`. This is quite important for code readability. 
```
Lets stick to PEP8 and not use camelcase